doc/getting-started/launching-a-testnet.md

```diff
@@ -0,0 +1,103 @@
+### Prerequisites
+You must already be able to build everything in the `cardano-node` repository.
+
+## Running the chairman test
+The chairman test will run some testnets and perform some basic tests to ensure they
+are working.
+
+To run the chairman tests, first build the necessary executables:
+
+```bash
+$ cabal build cardano-cli cardano-node cardano-node-chairman cardano-testnet
+```
+
+Then run the tests:
+
+```bash
+cabal test cardano-node-chairman
+```
+
+## Launching a testnet
+To create a testnet, first build the necessary executables:
+
+```bash
+$ cabal build cardano-cli cardano-node cardano-node-chairman cardano-testnet
+```
+
+Then get a list of available testnets to run:
+
+```bash
+$ cd cardano-node-chairman
+$ cabal run cardano-testnet
+Usage: cardano-testnet COMMAND
+
+Available options:
+  -h,--help                Show this help text
+
+Commands:
+  byron-shelley
+  shelley
+```
+
+Then create the testnet.  For example:
+
+```bash
+$ cabal run cardano-testnet byron-shelley
+  ✗ <interactive> failed at testnet/Testnet/Run.hs:34:3
+    after 1 test.
+
+        ┏━━ src/Testnet/ByronShelley.hs ━━━
+     68 ┃ testnet :: H.Conf -> H.Integration [String]
+     69 ┃ testnet H.Conf {..} = do
+     70 ┃   -- This script sets up a cluster that starts out in Byron, and can transition to Shelley.
+     71 ┃   --
+...
+       ┏━━ testnet/Testnet/Run.hs ━━━
+    25 ┃ testnetProperty :: (H.Conf -> H.Integration ()) -> H.Property
+    26 ┃ testnetProperty tn = H.integration . H.runFinallies . H.workspace "chairman" $ \tempAbsPath' -> do
+       ┃ │ Workspace: /private/var/folders/zh/ln41q4zs52x2fd61rxccmq640000gn/T/chairman/test-acaaa345c8802769
+    27 ┃   conf@H.Conf {..} <- H.mkConf tempAbsPath' 42
+    28 ┃
+    29 ┃   -- Fork a thread to keep alive indefinitely any resources allocated by testnet.
+    30 ┃   void . liftResourceT . resourceForkIO . forever . liftIO $ IO.threadDelay 10000000
+    31 ┃
+    32 ┃   void $ tn conf
+    33 ┃
+    34 ┃   H.failure -- Intentional failure to force failure report
+       ┃   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    This failure can be reproduced by running:
+    > recheck (Size 0) (Seed 7516688192894072998 17634780818781912383) <property>
+
+Testnet is running.  Type CTRL-C to exit.
```

Comment:
The output is a bit confusing to me: it shows a failed test, however the testnet seems to be running fine. It'd be nice to have an explanation on this.

Reply:
This is a result of using the `hedgehog` library to do the test report layout.  There is currently no way of trigging a detailed test report except by way of failure.  Some changes will need to be pushed upstream to the `hedgehog` library to fix this.

Reply:
I see, so maybe it'd be nice to clarify this in the documentation, since I can imagine readers being confused about this. I think you mentioned this later on, but it'd be nice to mention earlier the fact that the reader should not worry about the failure and that it will be explained later on.

Reply:
This is why there is a comment in the test output to that effect:

```
-- Intentional failure to force failure report
```

I think if the message there isn't obvious enough then it's better to fix that message.

Reply:
I added some more details.